os_migrate/roles/export_user_project_role_assignment/tasks/main.yml

```diff
@@ -0,0 +1,112 @@
+- name: scan available roles
```

Comment:
We'll want to only scan and export role assignments here. It seems the role assignment scanning module is not yet implemented so we'll have to implement it ourselves. We already did this for some resources:

https://github.com/os-migrate/os-migrate/blob/main/os_migrate/plugins/modules/os_keypairs_info.py

https://github.com/os-migrate/os-migrate/blob/main/os_migrate/plugins/modules/os_routers_info.py

https://github.com/os-migrate/os-migrate/blob/main/os_migrate/plugins/modules/os_security_groups_info.py